elasticsearch.md

```diff
@@ -799,11 +799,108 @@ So that Kibana will forward the necessary headers to Elasticsearch.
 
 [Impersonation](details/impersonation.md) is supported by this rule without an extra configuration.
 
-#### `groups_or` (or `groups`)
+#### Groups subrules
 
-`groups_or: ["group1", "group2"]`
+The ACL block will match, when the user belongs to groups matching the specified conditions.
 
-The ACL block will match when the user belongs to any of the specified groups. The information about what users belong to what groups is defined in the `users` section, typically situated after the ACL, further down in the YAML.
+The groups subrules use the user definitions from the `users` section. In that section, we define static users (and we assign groups to them) or we can authorize dynamic users (and we can map the external groups to the local groups).
```

Comment:
1. subrules -> rules
2. let's add https://docs.readonlyrest.com/elasticsearch#users-and-groups link to "the `users` section"